tools/fuzzilli/fuzzilli.cpp

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <cstring>
+#include <hermes/hermes.h>
+#include <jsi/jsi.h>
+using facebook::hermes::HermesRuntime;
+using facebook::hermes::makeHermesRuntime;
+using facebook::jsi::HostObject;
+using facebook::jsi::JSIException;
+using facebook::jsi::Object;
+using facebook::jsi::PropNameID;
+using facebook::jsi::Runtime;
+using facebook::jsi::String;
+using facebook::jsi::StringBuffer;
+using facebook::jsi::Value;
+
+//
```

Comment:
Is this body of code some well-known thing? I have no idea what it's doing or why.
Are there any libraries that provide this for us like something clang provides?

Reply:
This provides the coverage info. It is adapted from Google V8.

Reply:
I've made it clearer.